src/node_jsvmapi_types.h

```diff
@@ -51,6 +50,7 @@ enum napi_valuetype {
   napi_symbol,
   napi_object,
   napi_function,
+  napi_external,
```

Comment:
An external is not an object. Think of it as just a `void*` that can be passed around as a JavaScript value. There is generally no need to attach external data to arbitrary JS Objects, and neither V8 nor Chakra can support that.